pkg/validate/raw/validate/namespace_validator.go

```diff
@@ -27,7 +27,8 @@ import (
 // to the following rules:
 // - if the object is a Namespace, it must not be `config-management-system`
 // - if the object has a metadata namespace, it must be a valid k8s namespace
-//   and it must not be in `config-management-system`
+//
```

Comment:
nit: remove the empty line to keep the diff only relevant to the changes.

Reply:
this seems to happen automatically when i ran `make fmt-go`, will keep an eye out next time